src/main/java/io/vertx/codegen/annotations/DataObject.java

```diff
@@ -81,4 +81,9 @@
    */
   Class<? extends Case> jsonPropertyNameFormatter() default LowerCamelCase.class;
 
+  /**
+   * @return true if buffers codec should default to base64 basic encoding.
+   * Default {@code false}, use the platform default.
+   */
+  boolean base64BasicBuffers() default false;
```

Comment:
do we have names for the 2 kinds of encoder we provide? e.g what you call platform default above could be named `RFC4648` or `Basic` (the javadoc of `Base64` says _using the Basic type base64 encoding scheme_)